go/teams/chain.go

```diff
@@ -1375,6 +1390,31 @@ func (t *TeamSigChainPlayer) addInnerLink(
 		res.newState = prevState.DeepCopy()
 		t.updateInvites(&res.newState, additions, cancelations)
 		return res, nil
+	case libkb.LinkTypeSettings:
+		err = libkb.PickFirstError(
+			allowInflate(false),
+			hasPrevState(true),
+			hasName(false),
+			hasMembers(false),
+			hasParent(false),
```

Comment:
subteams scan't have settings?

Reply:
also should ban completedinvites

Reply:
Subteams can have settings, and it's even tested in team_open_test.go. is something wrong here?

Reply:
Nevermind, this is good. I forgot that most links don't have `parent`